MinPlatformPkg/Test/Library/TestPointCheckLib/TestPointImageDump.h

```diff
@@ -0,0 +1,31 @@
+/** @file
+
+Copyright (c) 2017, Intel Corporation. All rights reserved.<BR>
+Copyright (c) Microsoft Corporation.
+SPDX-License-Identifier: BSD-2-Clause-Patent
+
+**/
+
+#include <Uefi.h>
```

Comment:
It would be a little cleaner to try to reduce the includes in these header files to only what they need. Then let the C files account for their own dependencies as they need.

For now, can you please check the new files added (like this one) to try to minimize the includes for their content?

Reply:
Good idea.  Fixed.